### ISSUE TITLE
error when building tiler

### ISSUE THREAD

**Issue by zimirrr** [Geovation/tiler#24]
Hello, 
I am having an issue while building container. I have win 7, installed "Docker Toolbox". 
When I try to do
`docker build -t tiler .`
it seems to exit with error
![error](https://user-images.githubusercontent.com/1256802/27800933-3e18440c-6024-11e7-82d0-dd30165aac76.png)

Can You point me on my error?

Best regards.

**Comment by JamesMilnerUK** [comment]
I think this has something to do with the stable version of Debian changing. I'm experimenting with moving from `debian:stable` to `debian:jessie`

**Comment by JamesMilnerUK** [comment]
I'm pushing a fix to the 'fix' branch. If it passes CI I will merge into master and that should work for you :)

**Comment by JamesMilnerUK** [comment]
@zimirrr this should be fixed now. Let me know how you get on.

**Comment by zimirrr** [comment]
Still doesn't work, here is log file, that I made when building
[log.txt](https://github.com/Geovation/tiler/files/1127008/log.txt)
I hope it helps to debug.
Best regards.

**Comment by JamesMilnerUK** [comment]
@zimirrr the docker build works on my machine and also on the CI server. Which operating system are you using? 

Thanks for posting the log file. By the looks of it there is an issue with you connecting to GitHub to clone. Potentially a network/firewall issue. I did some research and it appears: https://github.com/angular/angular-phonecat/issues/141#issuecomment-40481120 has a solution, which is to move over to HTTPS rather that the using git.

Interestingly I noticed a bigger win, which is we don't use Mbutil anymore! So we can just remove it.
